source/change-streams/change-streams.rst

```diff
@@ -278,6 +278,16 @@ Driver API
      * @note this is an option of the `$changeStream` pipeline stage.
      */
     startAtOperationTime: Optional<Timestamp>;
+
+    /**
+     * Similar to 'resumeAfter', this option takes a resume token and starts a
+     * new change stream returning the first notification after the token.
+     * This will allow users to watch collections that have been dropped and recreated
+     * or newly renamed collections without missing any notifications.
+     * @since 4.2
+     * @see https://docs.mongodb.com/master/changeStreams/#change-stream-start-after
+     */
+     startAfter: Optional<Timestamp>
```

Comment:
I think type should be `Optional<Document>`, since it's a resume token (like `resumeAfter`, not `startAtOperationTime`). Also trailing `;` for consistency.